test/prism_regression/begin.rb

```diff
@@ -0,0 +1,6 @@
+# typed: true
+
+begin
+  foo
+# ^^^ error: Method `foo` does not exist on `T.class_of(<root>)`
```

Comment:
You can use a string or integer literal, so you won't have to deal with this error

```suggestion
  1
  2
  3
```